packages/twenty-server/src/modules/attachment/repositories/attachment.repository.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+
+import { EntityManager } from 'typeorm';
+
+import { WorkspaceDataSourceService } from 'src/engine/workspace-datasource/workspace-datasource.service';
+import { StorageDriverType } from 'src/engine/integrations/file-storage/interfaces';
+
+@Injectable()
+export class AttachmentRepository {
```

Comment:
Not sure whether this was expected to be added or not. Let me know if there is any other way how to persist the attachment entity :)

Reply:
Yes ! It was needed indeed :)